### DISCUSSION TITLE
Sticky Footer ScrollView

### DISCUSSION THREAD

**Discussion by bl1nch** [flet-dev/flet#5527] [Extensions]
### Duplicate Check

- [X] I have searched the [opened issues](https://github.com/flet-dev/flet/issues) and there are no duplicates

### Describe the requested feature

It is not possible to create a widget that will snap to the end of the page if the height of the page content is less than the height of the window, but will also scroll with the content if there is not enough space.

This issue occurs because any `Control` with `expand = true` in a `scrollable Column` becomes infinite.

Example in Flutter: https://pub.dev/packages/sticky_footer_scrollview

### Screenshots

![demo1](https://github.com/flet-dev/flet/assets/130155870/e78fa4bf-0066-4daf-911f-ee4e9dfda66a)
![demo3](https://github.com/flet-dev/flet/assets/130155870/957ae587-cc7f-41ed-b43a-51b5cd275d2e)

**Comment by ndonkoHenri** [comment]
Example of sticky footer:
```py

import flet as ft


def main(page: ft.Page):
    page.padding = ft.Padding(10, 10, 10, 0)  # remove the bottom padding
    page.add(
        ft.Column(
            [
                ft.ListTile(
                    title=ft.Text(f"Item {i}"),
                    bgcolor=ft.colors.GREEN_ACCENT_400,
                )
                for i in range(50)
            ],
            scroll=ft.ScrollMode.ALWAYS,
            expand=True,
        ),
        ft.ListTile(title=ft.Text("I stick to the bottom!"), bgcolor=ft.colors.BLUE),
    )


ft.app(main)

```

**Comment by bl1nch** [comment]
> Example of sticky footer:
> 
> ```python
> import flet as ft
> 
> 
> def main(page: ft.Page):
>     page.padding = ft.Padding(10, 10, 10, 0)  # remove the bottom padding
>     page.add(
>         ft.Column(
>             [
>                 ft.ListTile(
>                     title=ft.Text(f"Item {i}"),
>                     bgcolor=ft.colors.GREEN_ACCENT_400,
>                 )
>                 for i in range(50)
>             ],
>             scroll=ft.ScrollMode.ALWAYS,
>             expand=True,
>         ),
>         ft.ListTile(title=ft.Text("I stick to the bottom!"), bgcolor=ft.colors.BLUE),
>     )
> 
> 
> ft.app(main)
> ```

Sorry but that's not exactly what I meant. Sticky footer should also scroll with the content in the column if there is not enough free space on the page, and should only stick to the bottom if there is free space between the content in the column and the footer

**Comment by bl1nch** [comment]
Are you still planning to introduce `Sliver` controls in `v1` or have you decided to push it to later releases?
